### DISCUSSION TITLE
PatchTool 6.3 1/11

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#268] [Announcements]
國際服6.3的相容性更新，漢化內容主要更新至6.20。

部分內容，例如技能改動，漢化更新至6.3。

檔案：

- FFXIVChnTextPatch-20230111.rar：本體。一般而言，下載這個即可。JRE縮小過。

選用：

- rawexd-zhcn-20230111.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- backup-20230111.rar：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

目前測試，在之前的更新後，第三方4k字體包可能會導致遊戲崩潰。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.72.0'>PatchTool 6.3 1/11</a>.</em>

**Comment by ShikiNatsumeYuzu** [comment]
收到，感谢作者大大，辛苦了

**Comment by qwe3410475** [comment]
感謝作者更新，辛苦了！

**Comment by Kyo0829** [comment]
感謝作者，辛苦了~

**Comment by tonyjediex** [comment]
大哥效率真高 辛苦了!

**Comment by cs5886147** [comment]
辛苦了，感謝你無私的貢獻
